docs/docs-components/SandpackExample.js

```diff
@@ -94,7 +102,7 @@ export default function SandpackExample({
   code: ?string | (() => Node),
   layout?: 'row' | 'column',
   name: string,
-  previewHeight?: number,
+  previewHeight?: 'sm' | 'md' | number,
   showEditor?: boolean,
```

Comment:
I think we need an option to hide all the buttons for the "Don't" examples

Reply:
DONE